assets/images/general/collection_mascot.svg

```diff
@@ -1,43 +1,218 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-
-<svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   version="1.1"
-   id="mouth"
-   x="0px"
-   y="0px"
-   viewBox="0 0 384.5 384.5"
-   enable-background="new 0 0 384.5 384.5"
-   xml:space="preserve"
-   inkscape:version="0.91 r13725"
-   sodipodi:docname="collection_otter.svg"><metadata
-     id="metadata127"><rdf:RDF><cc:Work
-         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
-           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /></cc:Work></rdf:RDF></metadata><defs
-     id="defs125" /><sodipodi:namedview
-     pagecolor="#ffffff"
-     bordercolor="#666666"
-     borderopacity="1"
-     objecttolerance="10"
-     gridtolerance="10"
-     guidetolerance="10"
-     inkscape:pageopacity="0"
-     inkscape:pageshadow="2"
-     inkscape:window-width="640"
-     inkscape:window-height="480"
-     id="namedview123"
-     showgrid="false"
-     inkscape:zoom="0.61378414"
-     inkscape:cx="192.25"
-     inkscape:cy="192.25"
-     inkscape:window-x="0"
-     inkscape:window-y="0"
-     inkscape:window-maximized="0"
-     inkscape:current-layer="mouth" /></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
No need to change this image (but we can use a better placeholder in develop, if you like).